docs/docs/advanced/debugging.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_position: 10
+title: 🐛 Debugging
+---
+
+In some cases, you'll want to debug your Dart Frog app, and you might have noticed that by adding just the breakpoint, the debugger doesn't stop there like it does by default in a Flutter app.
+Below, there are two quick and easy options for debugging code in Dart Frog.
+
+## Debugging with Dart Frog Plugin 🐸
+
+:::caution
+This plugin is only available for Visual Studio Code, if you are using another IDE, please check the second option.
+:::
```

Comment:
Maybe we can add another note for those interested in an Android Studio extension to refer to the issue: https://github.com/VeryGoodOpenSource/dart_frog/issues/1326, or for those interested in OpenVSX, https://github.com/VeryGoodOpenSource/dart_frog/issues/907.